source/_data/plugins.yml

```
@@ -395,7 +395,7 @@
       keywords: [component, vue, vue.js]
 
 - name: Visual Testing
-  description:
+  description: Visual testing is a great complement to functional testing, read our [full visual testing guide](https://on.cypress.io/visual-testing)
```

Use url tags to verify we are not linking to dead links.

```suggestion
  description: Visual testing is a great complement to functional testing, read our {% url "full visual testing guide" visual-testing %}
```